packages/fuels/Cargo.toml

```diff
@@ -56,5 +56,4 @@ std = [
 ]
 # TODO: To be removed once https://github.com/FuelLabs/fuels-rs/issues/881 is unblocked.
 test-type-paths = []
-fuel-core-lib = ["fuels-test-helpers?/fuel-core-lib", "dep:fuel-core"]
-rocksdb = ["fuel-core?/rocksdb"]
+fuel-core-lib = ["fuels-test-helpers?/fuel-core-lib", "fuel-core"]
```

Comment:
If you don't say `dep:` then you're implicitly making another feature flag called "fuel-core". 

Having both `fuel-core-lib` and `fuel-core` is confusing.